Content.Shared/StatusEffect/StatusEffectsSystem.cs

```diff
@@ -144,6 +171,7 @@ private void OnHandleState(EntityUid uid, StatusEffectsComponent component, ref
             // and keep the relevant component the same.
             if (HasStatusEffect(uid, key, status))
             {
+                cooldown.Item2 += status.ActiveEffects[key].Cooldown.Item2 - status.ActiveEffects[key].Cooldown.Item1;
                 status.ActiveEffects[key] = new StatusEffectState(cooldown, status.ActiveEffects[key].RelevantComponent);
```

Comment:
I assume this is mean to work like the `TryAddTime` function? Currently it not only adds time, but also resets the already elapsed time. I.e., if you add 1 second of an effect that's already been going for 9.9/10 seconds, it will continue for another 11 seconds. 

I guess this should probably just use the same code as TryAddTime, unless you want the cooldown meter on the alert to stay at 100% until the reagent stops adding time. In that case I think `status.ActiveEffects[key].Cooldown.Item1` here just needs to be replaced with the current time?

Also the comment above this bit of code is outdated, it still indicates that the time is just set instead of added to.